apache/yunikorn-k8shim
# [YUNIKORN-1755] update deployments/examples/spark/

### What is this PR for?
Update Spark on K8s with YuniKorn example:
1. Change the default version to: spark version is: 3.3.1, default hadoop version is: 3.
2. `SPARK_EXAMPLE_JAR` and `SPARK_DOCKER_IMAGE` update.
3. After version 3.2.1, there is no need to format the official checksum file.
4. Create a namespace, service account and role binding according to [yunikorn web](https://yunikorn.apache.org/docs/user_guide/workloads/run_spark).
5. update spark-submit according to [yunikorn web](https://yunikorn.apache.org/docs/user_guide/workloads/run_spark).

### What type of PR is it?

### Todos

### What is the Jira issue?
https://issues.apache.org/jira/projects/YUNIKORN/issues/YUNIKORN-1755

### How should this be tested?
1. install yunikorn
2. `bash ./run.sh -h 3 -s 3.3.1` and `bash ./run.sh -h 3.2 -s 3.2.1`
3. check yunikorn web

### Screenshots (if appropriate)

### Questions:
